### PR TITLE
Added missing quotes in the babel config

### PR DESCRIPTION
**Pull request by PieterBoeren**
I noticed some missing quotes in the generated babel config, added those.

### CODE REVIEW

File: .changeset/wise-birds-lick.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"generator-single-spa": minor
```
Comment (joeldenning) [inline .changeset/wise-birds-lick.md:2]: This should be a patch, not a minor

**Review by joeldenning — COMMENTED** [review]
Babel config files are not strict json, so the quotes aren't necessary. Is there something the quotes help with? Maybe with syntax highlighting in VS Code, which might not realize that babel configs can be JSON 5?

**Comment by PieterBoeren** [comment]
It was indeed VS Code which made it quite red, but also the rest of the options in the file are with quotes, so for me, it's more consistent. Let me know if you agree, then I will fix the changes tomorrow.

**Comment by joeldenning** [comment]
Sure, we can release this as a patch - please address the comment above and then I'll approve and merge.

**Comment by joeldenning** [comment]
Released in generator-single-spa@4.0.2. See https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/CHANGELOG.md#402

**Pull request merged**